src/fs.c

```diff
@@ -761,7 +761,13 @@ void FS_InitFilesystemEx( qbool guess_cwd ) {
 #ifdef _WIN32
 		strlcat(com_homedir, "/ezQuake", sizeof(com_homedir));
 #else
-		strlcat(com_homedir, "/.ezquake", sizeof(com_homedir));
+		xdg = getenv("XDG_DATA_HOME");
+		if(xdg) {
+			strlcpy(com_homedir, xdg, sizeof(com_homedir));
+			strlcat(com_homedir, "/ezquake", sizeof(com_homedir));
+		} else {
+			strlcat(com_homedir, "/.local/share/ezquake", sizeof(com_homedir));
```

Comment:
I think many won't like this change. Why the '.local/share' prefix?

Reply:
if XDG_DATA_HOME isn't set, it should default to ~/.local/share

Reply:
> if XDG_DATA_HOME isn't set, it should default to ~/.local/share

I know ;).
But that is exactly why I commented...

Why do you think we need to move the ezquake home directory from "~/.ezquake" to "~/.local/share/ezquake"?
What makes you think that this is better, compared to the current one?
Is this your personal taste, or some kind of "standard" of some linux distro? If yes, which distro? What about other distro's "standard".

You see my point?


Also, as I have indicated, there is another PR open which is tackling these directories under linux. That PR seems to cover much more on top of the "home directory". So I rather wait on that PR to be finalized, instead of just merging this PR that only touches to home directory of ezquake.

Reply:
> Why do you think we need to move the ezquake home directory to '~/.local/share/ezquake'?
> What makes you think that this is better, compared to the current one?

it follows the XDG spec more closely :P

> I rather wait on that PR to be finalized

then have fun waiting, personally i think we will not hear from the author of that pr anytime soon

sorry for the late reply and i guess this pr can be closed then